serpentTools/parsers/microxs.py

```diff
@@ -182,7 +182,7 @@ def _precheck(self):
                               "Parser: {}\nFile: {}".format(self,
                                                             self.filePath))
             for tline in fid:
-                if 'NFY' in tline:
+                if 'NFY' in tline or 'XS_' in tline:
                     return
             raise SerpentToolsException("Fission yields values were not "
                                         "found in {}".format(self.filePath))
```

Comment:
Only recommendation would be changing the exception here to reflect missing fission yields or cross sections. But I'll defer to review and potential approval by @DanKotlyar

Reply:
I think this change does make sense,

Reply:
@aalfonsi bump

Reply:
addressed comment.